### PR TITLE
add more info to README, update Cartfile

### PR DESCRIPTION
**Pull request by stevemar**
* Update to 2.0.2 of SDK
* Add gif of simulator
* Rejigged the README a bit to make it easier to read
   * added prereqs
   * added more explanatory text
   * removed bit about modifying cartfile since it's already there by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,120 @@
...
 
 ![Credentials](docs/images/vizrec-creds.gif)
 
-Add your Visual Recognition API key to the [`Credentials.plist`](https://github.com/IBM/watson-visual-recognition-ios/blob/master/Core%20ML%20Vision/Credentials.plist):
+#### [`Credentials.plist`](https://github.com/IBM/watson-visual-recognition-ios/blob/master/Core%20ML%20Vision/Credentials.plist)
+
 ```plist
 <key>apiKey</key>
-<string>YOUR_API_KEY</string>
+<string>Rnqw721s...0m8aPu</string>
```
Comment (sanjeevghimire) [inline README.md]: should this be API_KEY?

**Review by sanjeevghimire — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stevemar) [reply README.md]: 🤷‍♂ i like giving the user an idea of what to expect for the structure of a key string, they can hopefully tell it's the API key given that the label says `apikey`. i don't think it matters much. If you want it back the other way, I can revert.

Comment (bourdakos1) [reply README.md]: I was going to ask the same

Comment (sanjeevghimire) [reply README.md]: I am fine with both.

Comment (bourdakos1) [reply README.md]: I've had a lot of people blindly copy exactly what I put in the examples, so I like to use as verbose of a "REPLACE_ME" as possible

**Review by bourdakos1 — APPROVED** [review]

**Review by bourdakos1 — APPROVED** [review]

**Pull request merged**